rotkehlchen/globaldb/handler.py

```diff
@@ -1180,6 +1181,76 @@ def get_historical_price_data(source: HistoricalPriceOracle) -> List[Dict[str, A
                  'to_timestamp': entry[3],
                  } for entry in query]
 
+    @staticmethod
+    def set_requests_cache_value(
+            cache_type: RequestsCacheType,
```

Comment:
Also not sure about the cache purpose. Should it be only `requests` cache or cache for everything? (so maybe rename it to something like `general_cache` ?